docs/ldml/tr35-keyboards.md

```diff
@@ -1316,11 +1324,16 @@ _Attribute:_ `form` (required)
 > This attribute specifies the physical layout of a hardware keyboard,
 > or that the form is a `touch` layout.
 >
-> It is recommended to always have at least one hardware (non-touch) form.
+> When using an on-screen keyboard, if there is not a `<layers form="touch">`
+> element, the hardware elements can be used for on-screen use. 
 > If there is no `hardware` form, the implementation may need
 > to choose a different keyboard file, or use some other fallback behavior when using a
 > hardware keyboard.
 >
+> Because a hardware keyboard facilitates non-trivial amounts of text input,
+> it is recommended to always have at least one hardware (non-touch) form, unless a layout
+> will only be used in touch form.
```

Comment:
The "unless" here doesn't seem necessary. Might be helpful to note that Android and iOS allow pairing of a hardware keyboard.

```suggestion
> Because a hardware keyboard facilitates non-trivial amounts of text input,
> and many touch devices can also be paired with a hardware keyboard, it 
> is recommended to always have at least one hardware (non-touch) form.
```

Reply:
sounds good (going with 'connected' as a more general instead of 'paired' even though that is technically the common case)